src/youtube-player/youtube-player.html

```diff
@@ -0,0 +1 @@
+<div #youtube_container></div>
```

Comment:
Instead of adding a `<div>` here, could the player just use its own host element? The template could then be empty.

Reply:
The div actually gets completely replaced by the iframe, which I think would be dangerous if we were to destroy the host element.

End result:
<youtube-player>
<iframe frameborder="0" allowfullscreen="1" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" title="YouTube video player" width="640" height="360" src="https://www.youtube.com/embed/PRQCAL_RMVo?enablejsapi=1&amp;origin=http%3A%2F%2Fnathantate.irv.corp.google.com%3A4200&amp;widgetid=1" id="widget2"></iframe>
</youtube-player>

Reply:
You can wrap code blocks in triple-ticks like this:
`````markdown
```html
<my-html-goes-here></...>
```
`````